roles/matrix-dynamic-dns/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+# Whether dynamic dns is enabled
+matrix_dynamic_dns_enabled: "{{ matrix_dynamic_dns_domain_configurations is defined }}"
```

Comment:
This is smart, but all our optional roles require an explicit `_enabled: true` to enable them, so I feel like this is a little strange.

Also, given that `matrix_dynamic_dns_domain_configurations` is set to `[]` in `group_vars/matrix_servers`, I think this would currently make `matrix_dynamic_dns_domain_configurations is defined` yield `true` and enable this role for everyone by default.

I think it's best that we set `matrix_dynamic_dns_enabled` to `true` in the role itself (so that anyone who includes it into their own other playbook will have it enabled by default), and then `matrix_dynamic_dns_enabled: false` in `group_vars/matrix_servers` (so that our "distribution" would have it be an optional component, disabled by default).

Reply:
Done